### DISCUSSION TITLE
count(): Parameter must be an array when calling toArray()

### DISCUSSION THREAD

**Discussion by heliostr** [laravel/framework#40406] [Q&A]
My full eloquent:
`UsersReservations::where('check_in_time', DB::raw('CURDATE() + 1'))->with(['plans', 'camp.camp_owner'])->get()->toArray();`
If I not calling toArray(), it will be working normal to return the collection with item.
Please help !! My english not good so sorry. Thanks.

**Comment by stijn1989** [comment]
Just use the count method of the Collection class.

```php
$records = UsersReservations::where('check_in_time', DB::raw('CURDATE() + 1'))->with(['plans', 'camp.camp_owner'])->get();
$count = $records->count();
```

**Reply by Oxicode** [reply]
$count = $records->count() ?? 0;

**Reply by ousid** [reply]
If there are no records, the method returns 0 by default.
